plugins/destination/mongodb/go.mod

```diff
@@ -5,17 +5,20 @@ go 1.19
 require (
 	github.com/apache/arrow/go/v13 v13.0.0-20230509040948-de6c3cd2b604
 	github.com/cloudquery/plugin-pb-go v1.0.8
-	github.com/cloudquery/plugin-sdk/v2 v2.7.0
+	github.com/cloudquery/plugin-sdk/v3 v3.5.1
 	github.com/rs/zerolog v1.29.0
 	go.mongodb.org/mongo-driver v1.11.2
 )
 
 // TODO: remove once all updates are merged
 replace github.com/apache/arrow/go/v13 => github.com/cloudquery/arrow/go/v13 v13.0.0-20230509053643-898a79b1d3c8
 
+replace github.com/cloudquery/plugin-sdk/v3 => ../../../../plugin-sdk
```

Comment:
this still has to go as plugin-sdk [v3.6.0](https://github.com/cloudquery/plugin-sdk/releases/tag/v3.6.0) contains the https://github.com/cloudquery/plugin-sdk/pull/893 changes